client/src/pages/products/ProductPage.jsx

```diff
@@ -1,7 +1,44 @@
-import AddProduct from './AddProduct';
+import { useState, useEffect } from 'react';
+import { Button } from 'react-bootstrap';
+import ListOfItems from '../../components/ListOfItems';
 
 const ProductPage = () => {
-  return <AddProduct/>
+  const [products, setProducts] = useState([]);
+
+  // Funkcja do pobierania produktów
```

Comment:
I suggest using English language to add comments and naming other things in code to keep a consistency.